addons/binding/org.openhab.binding.sleepiq/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
I added this because I saw other bindings with it committed. If I remove it, Eclipse just generates it again so I would like to mark it ignored or have it in Git.

It sounds like it is OK to keep it from the discussion, but please let me know if you want me to delete it.